src/adapters/postgres/user-adapter.ts

```
@@ -1150,6 +1154,8 @@ export class PostgresUserService implements IServicelocator {
         userCreateDto.customFields &&
         userCreateDto.customFields.length > 0
       ) {
+        console.log('hi3');
```

ESLint

[error] Replace `'hi3'` with `"hi3"`

(prettier/prettier)